lib/challenge.js

```diff
@@ -51,7 +62,12 @@ var pet_owner;
 var some_pet;
 
 // Answer Starts Here
-
+var some_pet = {
+  name: 'Holly',
+  species: "Canis lupis Familiarus",
+  breed: 'Corgi',
+  makeNoise: function(bark)
```

Comment:
Very much on the right track. The function in the value part of this key-value pair is missing a body, though. If it had a body, you could put statements in the body, and could complete the question exactly.